lib/more_core_extensions/core_ext/array/deletes.rb

```diff
@@ -15,6 +15,15 @@ def delete_nils
     def delete_blanks
       delete_if { |i| i.blank? }
     end
+
+    # Deletes all keys and subkeys that match +key+.
+    #
+    # [{:a => {:b => 2, :c => 3}}].deep_delete(:b) # => [{:a => {:c => 3}}]
```

Comment:
Can you 2 space indent here similar to the others?

Reply:
Done!